FastSearch.UnitTests/SearchTests.cs

```diff
@@ -156,5 +156,60 @@ public void IndexAndSearchTest2()
             Assert.AreEqual(r1.Count, r2.Count, list.Count);
             Assert.AreEqual(r1.Count, r2.Count, union.Count);
         }
+        [TestMethod]
+        public void ExactMatchSearch()
+        {
+            var lines = new List<string>

```

Comment:
This does not have to be a [List](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1?view=net-6.0), this could be an array. Never use a `List<T>` when an array will do.